### PR TITLE
Sum/nansum numpy-like impl

### PR DESCRIPTION
**Pull request by 1e-to**

### CODE REVIEW

**Comment by 1e-to** [comment]
Perfomance: https://teamcity-or.intel.com/buildConfiguration/Sat_Sdc_Linux_SdcPy37numpy116_Testing_SdcBenchmarks?branch=570%2Fmerge&buildTypeTab=overview

**Comment by PokhodenkoSA** [comment]
> Perfomance: https://teamcity-or.intel.com/buildConfiguration/Sat_Sdc_Linux_SdcPy37numpy116_Testing_SdcBenchmarks?branch=570%2Fmerge&buildTypeTab=overview

Can not open and see results. Could you, please, attach results screenshot?

File: sdc/functions/numpy_like.py [hunk 1]
```diff
@@ -97,3 +97,27 @@ def sdc_astype_number_impl(self, dtype):
...
 
         return sdc_astype_number_impl
+
+
+@sdc_register_jitable
+def sum(a):
+    length = len(a)
+    result = 0
+    for i in prange(length):
+        if not numpy.isnan(a[i]):
```
Comment (AlexanderKalistratov) [inline sdc/functions/numpy_like.py]: Let's use here `get_isnan` @PokhodenkoSA  knows the details

Comment (PokhodenkoSA) [reply sdc/functions/numpy_like.py]: I think @AlexanderKalistratov means this https://github.com/IntelPython/sdc/pull/579#discussion_r376252981

**Comment by AlexanderKalistratov** [comment]
@1e-to conflict

**Comment by AlexanderKalistratov** [comment]
@1e-to conflict

**Pull request merged**